doc: update on cellular UDP sample (#12641)
nrfconnect/sdk-nrf: samples/cellular/udp/README.rst

   :local:
   :depth: 2

> The Cellular: UDP sample demonstrates the sequential transmission of UDP packets to a predetermined server identified by an IP address and a port.

Reviewer: ```suggestion
This sample demonstrates the sequential transmission of UDP packets to a predetermined server identified by an IP address and a port.
```

Author: The reason to explicitly mention the sample name as cellular: UDP is, we have a similar sample in the networking folder which has Wi-Fi support. https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/net/udp/README.html.

So, I think we could keep this as is?

Reviewer: I'd say we can still safely say "This sample", as the reader is possibly on this page when reading.

Author: Discussed offline, and decided to keep the sample name at the first instance.

You can configure the frequency with which the packets are transmitted and the size of the UDP payload through the Kconfig system.
In addition to setting of the above options, you can also set the various LTE parameters that are related to current consumption for adding low power behavior to the device.

> The Cellular: UDP sample can be used to characterize the current consumption of the nRF91 Series SiP.

Reviewer: ```suggestion
You can use this sample to characterize the current consumption of the nRF91 Series SiP.
```

Author: may be : _You can use  Cellular: UDP sample ...._ ?
